packages/client/src/modules/forms/Registration.js

```diff
@@ -0,0 +1,139 @@
+import { Button, Container, Grid, TextField, Typography } from "@mui/material";
+import { Checkbox } from "@mui/material";
+import { useFormik } from "formik";
+import * as yup from "yup";
+
+const initialValues = {
+  name: "",
+  email: "",
+  address: "",
+  city: "",
+  state: "",
+  zip: "",
+};
+
+const validationSchema = yup.object({
+  name: yup.string().required().label("Name"),
+  email: yup.string().required().label("Email"),
+  address: yup.string().required().label("Street Address"),
+  city: yup.string().required().label("City"),
+  state: yup.string().required().label("State"),
+  zip: yup.string().required().label("Zip Code"),
+  mailing: yup.string().required().label("Mailing List"),
```

Comment:
The checkbox opt-in is going to be either true or false, not a string. Because of the nature of the checkbox input, you can assume the value is false if it's not checked, so you don't actually validate it. Remove this and your form should validate without issue.